charts/jaeger/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: 1.22.0
 description: A Jaeger Helm chart for Kubernetes
 name: jaeger
 type: application
-version: 0.46.5
```

Comment:
(nitpick) I'd bump it to 0.47.0.

Reply:
Really? Currently, I can't deploy this chart to our cluster because of #277.

That's why I thought this change to be a bugfix rather than a feature, but I can change it if you want